### PR TITLE
DOCS-2290 Datadog-Amazon CloudFormation Edits

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Added a region banner to specify this integration is only supported in us-east-1, an additional command (set-type-configuration), and doc edits. 

[Doc Ticket](https://datadoghq.atlassian.net/browse/DOCS-2290)

### Description of the Change

Documentation updates.

### Review checklist (to be filled by reviewers)

- [ ] Feature or bug fix MUST have appropriate tests (unit, integration, etc...)
- [ ] PR title must be written as a CHANGELOG entry [(see why)](/CONTRIBUTING.md#pull-request-title)
- [ ] Files changes must correspond to the primary purpose of the PR as described in the title (small unrelated changes should have their own PR)
- [ ] PR must have one `changelog/` label attached. If applicable it should have the `backward-incompatible` label attached.
- [ ] PR should not have `do-not-merge/` label attached.
- [ ] If Applicable, issue must have `kind/` and `severity/` labels attached at least.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
+{{< site-region region="us3,eu,gov" >}}
+<div class="alert alert-warning">
+  This integration is only supported in the US East (us-east-1) region.
+</div>
+{{< /site-region >}}
+
```
Comment (zippolyte) [inline README.md:6]: The us-east-1 region is for customers own amazon account, not our regions. The integration can be used to send data to any of our DC.

Comment (zippolyte) [reply README.md:6]: Additionally, this region restriction is only for the public registry. Anybody can still use the zip packages we provide to privately register the extensions in any region.

File: README.md [hunk 2]
```diff
@@ -84,13 +90,22 @@ To get started:
       * See the [Resources Available section](#resources-available), which links to examples of the latest supported S3 links.
     * `VERSION_ID`: The underlying version of the resource as returned by the command in step `2`.
 
-4. In your AWS account, [create your AWS stack][3] that includes any of the registered Datadog resources.
+4. Set the newly registered resource configuration by running the following in your terminal:
+
+    ```shell
+    aws cloudformation set-type-configuration \
+        --type-name "<DATADOG_RESOURCE_NAME>" arn:aws:cloudformation:<OUR_REGION>:<OUR_ACCOUNT>:type/resource/Datadog-Monitors-Monitor \
```
Comment (zippolyte) [inline README.md:97]:
```suggestion
        --type-name "<DATADOG_RESOURCE_NAME>" \

**Review by zippolyte — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (alai97) [reply README.md:6]:
```suggestion
{{< site-region region="us3,eu,gov" >}}
<div class="alert alert-warning">
  This integration is only supported in the US East (us-east-1) region of the AWS Management Console UI.
</div>
{{< /site-region >}}

```

File: README.md [hunk 3]
```diff
@@ -1,8 +1,14 @@
+{{< site-region region="us3,eu,gov" >}}
+<div class="alert alert-warning">
+  This integration is only supported in the US East (us-east-1) region.
+</div>
+{{< /site-region >}}
+
 # Datadog-AWS CloudFormation
 
 [AWS CloudFormation][1] gives you templates to describe, configure, and provision all of the AWS resources in your environment at once. The Datadog-AWS CloudFormation Resources allow you to interact with the supported Datadog resources.
```
Comment (alai97) [inline README.md:9]:
```suggestion
[AWS CloudFormation][1] gives you templates to describe, configure, and provision all of the AWS resources in your environment at once. The Datadog-AWS CloudFormation Resources allow you to interact with the supported Datadog resources, send resources to any Datadog datacenter, and privately register an extension in any region with Datadog resources.
```

Comment (alai97) [reply README.md:9]: @zippolyte I added your comments to the site partial in this introductory paragraph. Does this content logically make sense?

File: README.md (hunk 1 above)
Comment (zippolyte) [reply README.md:6]: I don't think I would phrase it like this. More like "Datadog Cloudformation resources are only available on the cloudformation public registry in the us-east-1 region. You can still use the provided packages to privately register a resource in any region."
Also show this all the time, not only for `us3,eu,gov` as those are DD regions and it's unrelated to availability.
Does that make sense ?

Comment (alai97) [reply README.md:6]: @zippolyte one more time!

File: README.md [hunk 4]
```diff
@@ -1,8 +1,14 @@
+{{< site-region region="us3,eu,gov" >}}
```
Comment (zippolyte) [inline README.md:1]: I'm still concerned about this part, why us1 is excluded 🙂

Comment (alai97) [reply README.md:1]: I didn't commit my suggestions yet!

**Review by zippolyte — APPROVED** [review]

**Review by cswatt — APPROVED** [review]

**Pull request merged**